### DISCUSSION TITLE
Vercel Redeployment and Github

### DISCUSSION THREAD

**Discussion by waynemareci** [vercel/community#2399] [Help] [answered]
### Summary

Just curious - if I back out of a Vercel deployment using the Redeploy option, what happens to the associated source code on Github? Does it revert to the previous deployment? Is this behavior documented anywhere?

**Comment by amyegan** [comment, marked as answer]
The source code on GitHub is not changed by Vercel deployments or redeployment 🙂

**Reply by amyegan** [reply]
You can find more info on the GitHub deployment integration in our docs: [Deploying GitHub Projects with Vercel](https://vercel.com/docs/concepts/deployments/git/vercel-for-github#)
